pmdarima/model_selection/_validation.py

```diff
@@ -304,6 +322,14 @@ def cross_val_predict(estimator, y, exogenous=None, cv=None, verbose=0,
     for i, (pred_block, test_indices) in enumerate(prediction_blocks):
         pred_matrix[test_indices, i] = pred_block
 
+    if return_forecast_horizons:
+        pred_matrix = np.ones((y.shape[0], cv.horizon)) * np.nan
+        prediction_blocks = np.array(prediction_blocks)
+        predictions = prediction_blocks[:,0,:]
+        indices = prediction_blocks[:,1,:][:,0].astype(int) - 1
+        pred_matrix[indices] = predictions
+        return pred_matrix
```

Comment:
Would you mind adding a test case for this? Maybe assert, e.g.
* Expected shape
* Expected test size
* No nans in test forecasts

Reply:
Sure!

Reply:
btw if you're on a POSIX machine (not a windows), you can use the `Makefile` for some handy targets for local testing:
* `make test-lint` will run flake8
* `make test-unit` will run the unit tests